lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -57,9 +73,22 @@ export default function LMSGrader({
    * Makes an RPC call to the sidebar to change to the focused user.
    */
   const changeFocusedUser = useCallback(
-    /** @param {StudentInfo|null} user - The user to focus on in the sidebar */
-    user => clientRPC.setFocusedUser(user),
-    [clientRPC]
+    async (
+      /** @param {StudentInfo|null} user - The user to focus on in the sidebar */
+      user
+    ) => {
+      if (apiSync && user) {
+        const groups = await apiCall({
+          authToken,
+          path: apiSync.path,
+          data: { ...apiSync.data, gradingStudentId: user.lmsId },
```

Comment:
@marcospri Ah, I see that `lmsId` was added a couple of days ago on the backend:

https://github.com/hypothesis/lms/commit/dcefb1e3eeda913b6c6605d4c4beec8e4a83e48f

I'll need to update the front-end typing to correspond.

Reply:
Actually, can you help me understand what the `lmsId` _is_, @marcospri ? And how it differs from `userid`?

Reply:
The existing `userid` is the user on H while lmsId is the ID given to the user/student on the LMS (Blackboard here). We need the later to query for its groups on the group set while grading.